examples/metrics/src/main/java/com/google/cloud/opentelemetry/example/metrics/MetricsExporterExample.java

```diff
@@ -0,0 +1,74 @@
+package com.google.cloud.opentelemetry.example.metrics;
+
+import static java.util.Collections.singleton;
+
+import com.google.cloud.opentelemetry.metric.MetricConfiguration;
+import com.google.cloud.opentelemetry.metric.MetricExporter;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Random;
+
+public class MetricsExporterExample {
+  private static final Meter METER =
+      OpenTelemetry.getGlobalMeter("instrumentation-library-name", "semver:1.0.0");
+  private static final Random RANDOM = new Random();
+  private static io.opentelemetry.sdk.metrics.export.MetricExporter metricExporter;
+  private static IntervalMetricReader intervalMetricReader;
+
+  private static void setupMetricExporter() {
+    try {
+      MetricConfiguration configuration =
+          MetricConfiguration.builder().setDeadline(Duration.ofMillis(90000)).build();
```

Comment:
This is a pretty long deadline. Is that because of the issues on your laptop? I'm not sure if we need that long deadline, and example should probably just use `MetricExporter.createWithDefaultConfiguration()`, i.e. the default configuration should be good enough.